openmc/deplete/results.py

```diff
@@ -95,6 +95,60 @@ def from_hdf5(cls, filename: PathLike):
         )
         return cls(filename)
 
+    def get_activity(
+        self,
+        mat: typing.Union[Material, str],
+        units: str = "Bq/cm3",
+        by_nuclide: bool = False, 
+        volume: Optional[float] = None
+    ) -> Tuple[np.ndarray, np.ndarray]:
```

Comment:
The type hint here for the return value is deceiving, since the value could be a dictionary